TBAnnotationClustering-Swift/ViewController.swift

```diff
@@ -43,7 +43,7 @@ class ViewController: UIViewController, MKMapViewDelegate {
         let toRemove = NSMutableSet(set: before)
         toRemove.minusSet(after as Set<NSObject>)
         
-        dispatch_async(dispatch_get_main_queue()) { () -> Void in
+        NSOperationQueue().addOperationWithBlock() {
```

Comment:
Oh wait, it seems that I actually fixed this bug on the next commit:
https://github.com/thoughtbot/TBAnnotationClustering/pull/11/commits/be4bdcd3d80e114c42ebb161e24a54a3ec2ea410